src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/53.diff.sql

```diff
@@ -0,0 +1,1223 @@
+                    SET @CopiedSearchParams += @myRows
+
+                    INSERT INTO dbo.Tmp_TokenNumberNumberCompositeSearchParam 
+                             ( ResourceTypeId, ResourceSurrogateId, SearchParamId, SystemId1, Code1, CodeOverflow1, SingleValue2, LowValue2, HighValue2, SingleValue3, LowValue3, HighValue3, HasRange )
+                        SELECT ResourceTypeId, ResourceSurrogateId, SearchParamId, SystemId1, Code1, CodeOverflow1, SingleValue2, LowValue2, HighValue2, SingleValue3, LowValue3, HighValue3, HasRange
+                            FROM TokenNumberNumberCompositeSearchParam B
+                            WHERE ResourceTypeId = @ResourceTypeId 
+                            AND EXISTS (SELECT * FROM @SurrogateIds C WHERE C.ResourceSurrogateId = B.ResourceSurrogateId)
+                            AND NOT EXISTS (SELECT * 
+                                                FROM Tmp_TokenNumberNumberCompositeSearchParam A 
+                                                WHERE A.ResourceTypeId = @ResourceTypeId 
+                                                AND A.ResourceSurrogateId = B.ResourceSurrogateId)
+
+                    SET @myRows = @@rowcount
+                    EXECUTE dbo.LogEvent @Process=@ProcessName,@Status='Run',@Target='Tmp_TokenNumberNumberCompositeSearchParam',@Action='Insert',@Rows=@myRows
+                    SET @CopiedSearchParams += @myRows
+                END
+
+                UPDATE dbo.Parameters SET Char = @LastProcessed WHERE Id = @Id
+
+                IF datediff(second, @ReportDate, getUTCdate()) > 60
+                    BEGIN
+                        EXECUTE dbo.LogEvent @Process=@ProcessName,@Status='Run',@Target='Resource',@Action='Select',@Rows=@ProcessedResources,@Text=@LastProcessed
+                        EXECUTE dbo.LogEvent @Process=@ProcessName,@Status='Run',@Target='*SearchParam',@Action='Copy',@Rows=@CopiedSearchParams,@Text=@LastProcessed
+                        SET @ReportDate = getUTCdate()
+                        SET @ProcessedResources = 0
+                        SET @CopiedSearchParams = 0
+                    END
+            END
+
+            DELETE FROM @Types WHERE ResourceTypeId = @ResourceTypeId
+
+            SET @SurrogateId = 0
+        END
+
+        EXECUTE dbo.LogEvent @Process=@ProcessName,@Status='Run',@Target='Resource',@Action='Select',@Rows=@ProcessedResources,@Text=@LastProcessed
+        EXECUTE dbo.LogEvent @Process=@ProcessName,@Status='Run',@Target='*SearchParam',@Action='Copy',@Rows=@CopiedSearchParams,@Text=@LastProcessed
+
+        EXECUTE dbo.LogEvent @Process=@ProcessName,@Status='End',@Start=@cst
+    END TRY
+    BEGIN CATCH
+        IF error_number() = 1750 THROW -- Real error is before 1750, cannot trap in SQL.
+            EXECUTE dbo.LogEvent @Process=@ProcessName,@Status='Error';
+            THROW
```

Comment:
All 3 statement are on the same level. No indentation required.

Reply:
Fixed